include/osquery/query.h

```diff
@@ -92,10 +93,17 @@ Status deserializeRowJSONRJ(const std::string& json, Row& r);
  * @brief The result set returned from a osquery SQL query
  *
  * QueryData is the canonical way to represent the results of SQL queries in
- * osquery. It's just a vector of Row's.
+ * osquery. It's just a vector of Rows.
  */
 using QueryData = std::vector<Row>;
 
+/**
+ * @brief Set representation result returned from a osquery SQL query
+ *
+ * QueryDataSet -  It's set of Rows for fast search of a specific row.
+ */
+using QueryDataSet = std::multiset<Row>;
```

Comment:
This is a much better implementation. :)

Reply:
Thanks.